### ISSUE TITLE
tf_text_graph_faster_rcnn stops with error on faster_rcnn_nas_coco 

### ISSUE THREAD

**Issue by advanpix** [opencv/opencv#13720]
##### System information (version)
- OpenCV => 4.0.1
- Operating System / Platform => Windows 10 64 Bit
- Compiler => Visual Studio 2017

##### Detailed description

The tf_text_graph_faster_rcnn.py stops with error when trying to generate pbtxt file for faster_rcnn_nas model from TF object detection model zoo [1-2] following this guide [3].  

```
    Traceback (most recent call last):
      File "tf_text_graph_faster_rcnn.py", line 241, in <module>
        createFasterRCNNGraph(args.input, args.config, args.output)
      File "tf_text_graph_faster_rcnn.py", line 34, in createFasterRCNNGraph
        features_stride = float(config['feature_extractor'][0]['first_stage_features_stride'][0])
    KeyError: 'first_stage_features_stride'
```
Would appreciate any tips and ideas on how to fix this bug.

References.

[1] https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md

[2] https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/faster_rcnn_nas_coco.config

[3] https://github.com/opencv/opencv/wiki/TensorFlow-Object-Detection-API

**Comment by advanpix** [comment]
After #13743 the error has been fixed. Bow conversion completes successfully:

```
Number of classes: 90
Scales:            [0.25, 0.5, 1.0, 2.0]
Aspect ratios:     [0.5, 1.0, 2.0]
Width stride:      16.000000
Height stride:     16.000000
Features stride:   16.000000  
```

But OpenCV-4.0.1-dev fails to load the model:

```
OpenCV: terminate handler is called! The last OpenCV error is:
OpenCV(4.0.1-dev) Error: Unspecified error (Input [SecondStageFeatureExtractor/cell_14/prev_bn/gamma/read/_1324__cf__1330] for node [SecondStageFeatureExtractor/cell_14/prev_bn/FusedBatchNorm] not found) in cv::dnn::dnn4_v20190122
::<unnamed>::TFImporter::getConstBlob, file H:\Build\opencv-4.X-master-cuda-9-2-mkl-openmp\opencv-master\modules\dnn\src\tensorflow\tf_importer.cpp, line 535

```

**Comment by dkurt** [comment]
@advanpix, please try this changes: https://github.com/opencv/opencv/pull/14459

**Comment by advanpix** [comment]
Thank you very much for the update! 
I will be able to test it when it will be merged into master branch. I departed from 3.4 long time ago.

**Comment by dkurt** [comment]
@advanpix, Sure, thank you!

**Comment by farecomhalima** [comment]
Hello @advanpix , I can't understand the solution "After #13743 the error has been fixed." , did you fix the file "tf_text_graph_faster_rcnn " Thanks
